src/System.Net.Http/src/System/Net/Http/Unix/HttpClientHandler.Unix.cs

```diff
@@ -17,7 +17,7 @@ public virtual bool SupportsAutomaticDecompression
 
         public virtual bool SupportsProxy
         {
-            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            get { return true; }
```

Comment:
Should this also delegate to a SupportsProxy property on CurlHandler which then returns true?  With the current design of having CurlHandler be separate from HttpClientHandler, it seems like HttpClientHandler should contain zero logic other than containing a CurlHandler and delegating to it.